kwok/README.md

```
@@ -202,40 +121,19 @@ gh workflow run kwok-recipes.yaml -f recipe=your-recipe-name
 
 ## Troubleshooting
 
-### Pods stuck Pending
-
-Check tolerations match KWOK nodes:
+**Pods stuck Pending** — check tolerations and node selectors:
 ```bash
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)